apps/nextjs-website/src/components/atoms/ChatbotHistoryListItem/ChatbotHistoryListItem.tsx

```diff
@@ -34,10 +34,10 @@ const ChatbotHistoryListItem = ({ session }: ChatbotHistoryListItemProps) => {
   ).format(new Date(session.createdAt));
 
   return (
-    <ListItem>
+    <ListItem sx={{ paddingLeft: 0, width: '100%' }}>
```

Comment:
The text is too close to the edge with this hover padding value:
![Screenshot from 2024-10-07 15-50-07](https://github.com/user-attachments/assets/ee9aed7c-9acd-46bf-a8a9-1ed6c7f3d2df)